src/renderers/shared/stack/reconciler/ReactRef.js

```diff
@@ -56,20 +66,21 @@ ReactRef.shouldUpdateRefs = function(prevElement, nextElement) {
   // is made. It probably belongs where the key checking and
   // instantiateReactComponent is done.
 
-  var prevEmpty = prevElement === null || prevElement === false;
-  var nextEmpty = nextElement === null || nextElement === false;
-
   return (
     // This has a few false positives w/r/t empty components.
```

Comment:
Would be nice to learn what those false positives are since we're touching this code.